File: week_2/src/vel_pkg/scripts/vel_node.py
#!/usr/bin/env python3
#conding=utf-8

import rospy
from geometry_msgs.msg import Twist

if __name__ == "__main__":

    rospy.init_node("vel_node")

    pub = rospy.Publisher ("/cmd_vel",Twist,queue_size=10)
    r = rospy.Rate(30)

    vel_msg = Twist()
    vel_msg.linear.x = 0.1

    while not rospy.is_shutdown():
        pub.publish(vel_msg)
        r.sleep()

File: week_2/src/vel_pkg/src/vel_node.cpp
#include <ros/ros.h>
#include <geometry_msgs/Twist.h>

int main(int argc, char *argv[])
{

    ros::init(argc, argv ,"vel_node");
    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<geometry_msgs::Twist>("/cmd_vel",0);
    
    ros::Rate rate(30);

    geometry_msgs::Twist vel_msg;
    vel_msg.linear.x = 0.1;
    vel_msg.linear.y = 0;
    vel_msg.linear.z = 0;
    vel_msg.angular.x = 0;
    vel_msg.angular.y = 0;
    vel_msg.angular.z = 0;

    while(ros::ok)
    {
        pub.publish(vel_msg);
        ros::spinOnce();
        rate.sleep();
    }

    return 0;
}